recipe/meta.yaml

```
@@ -88,32 +88,54 @@ outputs:
         - {{ cdt('pciutils-devel') }}        # [linux]
         - {{ cdt('libxscrnsaver-devel') }}   # [linux]
         - {{ cdt('libxtst-devel') }}         # [linux]
+        - {{ cdt('libselinux') }}            # [linux]
         - {{ cdt('libselinux-devel') }}      # [linux]
         - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxdamage-devel') }}      # [linux]
         - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxfixes-devel') }}       # [linux]
         - {{ cdt('libxxf86vm') }}            # [linux]
-        - {{ cdt('mesa-libgbm') }}           # [linux]
-        - {{ cdt('pcre') }}                  # [linux and cdt_name != 'cos6']
-        - make                               # [not win]
+        - {{ cdt('libxxf86vm-devel') }}      # [linux]
+        - {{ cdt('libsm-devel') }}           # [linux]
+        - {{ cdt('libice-devel') }}          # [linux]
+        - {{ cdt('libxcb') }}                # [linux]
+        - {{ cdt('libxcb-devel') }}          # [linux]
+        - {{ cdt('expat-devel') }}           # [linux]
+        - {{ cdt('pcre') }}                  # [linux]
+        - {{ cdt('libglvnd-glx') }}          # [linux]
+        - {{ cdt('libglvnd-egl') }}          # [linux]
+        - {{ cdt('libxkbcommon-devel') }}    # [linux]
+        - {{ cdt('xcb-util-devel') }}        # [linux]
+        - {{ cdt('xcb-util-image-devel') }}  # [linux]
+        - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
+        - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
+        - {{ cdt('xcb-util-wm-devel') }}          # [linux]
         - jom                                # [win]
-        - pkg-config                         # [not win]
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - pyqt-builder                           # [build_platform != target_platform]
+        - sip                                    # [build_platform != target_platform]
+        - make
       host:
-        - {{ pin_subpackage('pyqt5-sip', exact=True) }}
-        - {{ cdt('mesa-libgl-devel') }}      # [linux]
-        - alsa-lib                           # [linux]
-        - dbus                               # [not win]
-        # one unconstrained to pick up global pinning + a lower bound
-        - qt
-        - qt >=5.12.9
         - python
+        - pip
+        - sip                                    # [build_platform == target_platform]
+        - toml
+        - pyqt-builder                           # [build_platform == target_platform]
+        - packaging
+        - qt-main 5.15.2
+        - {{ cdt('mesa-libgl-devel') }}          # [linux]
+        - {{ cdt('mesa-libegl-devel') }}         # [linux]
       run:
         - python
-        - {{ pin_compatible("qt", max_pin="x.x") }}
+        # - {{ pin_compatible("qt-main", max_pin="x.x") }}
+        - qt-main 5.15.2
```

```suggestion
        - qt-main 5.15
```

You should be able to pin runtime a little looser